-1 
Tested at: e7597c4ca37ddc400268361670cb989135bd3ec7
 I found an error when building: 
 
 <pre>>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/DuplicateListMerger.cc 
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/DuplicateTrackMerger.cc 
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/MultiTrackSelector.cc 
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/SimpleTrackListMerger.cc 
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/DuplicateListMerger.cc: In member function 'virtual void {anonymous}::DuplicateListMerger::produce(edm::StreamID, edm::Event&, const edm::EventSetup&) const':
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/DuplicateListMerger.cc:259:35: error: no match for 'operator|' (operand types are 'std::bitset<46ul>' and 'reco::TrackBase::TrackAlgorithm')
     otk.setAlgoMask(algoMask[isel]|reco::TrackBase::duplicateMerge);
                                   ^
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-11-25-2300/src/RecoTracker/FinalTrackSelectors/plugins/DuplicateListMerger.cc:259:35: note: candidates are:
In file included from /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/cms/cmssw/CMSSW_8_0_X_2015-11-25-2300/src/DataFormats/Math/interface/SIMDVec.h:55:0,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/cms/cmssw/CMSSW_8_0_X_2015-11-25-2300/src/DataFormats/GeometryVector/interface/Basic2DVector.h:3,
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12577/9961/summary.html

--- a/RecoParticleFlow/Benchmark/src/PFJetBenchmark.cc
+++ b/RecoParticleFlow/Benchmark/src/PFJetBenchmark.cc
@@ -316,6 +316,7 @@ void PFJetBenchmark::process(const reco::PFJetCollection& pfJets, const reco::Ge
 	unsigned int iter = 0; 
 	switch (trackRef->algo()) {
 	case TrackBase::ctf:
+        case TrackBase::duplicateMerge:
 	case TrackBase::initialStep:
 	  iter = 0;
 	  break;

--- a/RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporter.cc
+++ b/RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporter.cc
@@ -161,6 +161,7 @@ goodPtResolution( const reco::TrackRef& trackref) const {
   unsigned int Algo = 0; 
   switch (trackref->algo()) {
   case reco::TrackBase::ctf:
+  case reco::TrackBase::duplicateMerge:
   case reco::TrackBase::initialStep:
   case reco::TrackBase::lowPtTripletStep:
   case reco::TrackBase::pixelPairStep:

--- a/RecoParticleFlow/PFProducer/src/PFAlgo.cc
+++ b/RecoParticleFlow/PFProducer/src/PFAlgo.cc
@@ -1936,6 +1936,7 @@ void PFAlgo::processBlock( const reco::PFBlockRef& blockref,
       double blowError = 1.;
       switch (trackRef->algo()) {
       case TrackBase::ctf:
+      case TrackBase::duplicateMerge:
       case TrackBase::initialStep:
       case TrackBase::lowPtTripletStep:
       case TrackBase::pixelPairStep:
@@ -2397,6 +2398,7 @@ void PFAlgo::processBlock( const reco::PFBlockRef& blockref,
 
 	switch (trackref->algo()) {
 	case TrackBase::ctf:
+        case TrackBase::duplicateMerge:
 	case TrackBase::initialStep:
 	case TrackBase::lowPtTripletStep:
 	case TrackBase::pixelPairStep:

--- a/RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc
+++ b/RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc
@@ -2509,6 +2509,7 @@ unsigned int PFEGammaAlgo::whichTrackAlgo(const reco::TrackRef& trackRef) {
   unsigned int Algo = 0; 
   switch (trackRef->algo()) {
   case TrackBase::ctf:
+  case TrackBase::duplicateMerge:
   case TrackBase::initialStep:
   case TrackBase::lowPtTripletStep:
   case TrackBase::pixelPairStep:

--- a/RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc
+++ b/RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc
@@ -365,6 +365,7 @@ unsigned int PFEGammaFilters::whichTrackAlgo(const reco::TrackRef& trackRef) {
   unsigned int Algo = 0; 
   switch (trackRef->algo()) {
   case TrackBase::ctf:
+  case TrackBase::duplicateMerge:
   case TrackBase::initialStep:
   case TrackBase::lowPtTripletStep:
   case TrackBase::pixelPairStep:

--- a/RecoParticleFlow/PFProducer/src/PFElectronAlgo.cc
+++ b/RecoParticleFlow/PFProducer/src/PFElectronAlgo.cc
@@ -2675,6 +2675,7 @@ unsigned int PFElectronAlgo::whichTrackAlgo(const reco::TrackRef& trackRef) {
   unsigned int Algo = 0; 
   switch (trackRef->algo()) {
   case TrackBase::ctf:
+  case TrackBase::duplicateMerge:
   case TrackBase::initialStep:
   case TrackBase::lowPtTripletStep:
   case TrackBase::pixelPairStep:

--- a/RecoParticleFlow/PFTracking/plugins/PFElecTkProducer.cc
+++ b/RecoParticleFlow/PFTracking/plugins/PFElecTkProducer.cc
@@ -477,6 +477,7 @@ bool PFElecTkProducer::isFifthStep(reco::PFRecTrackRef pfKfTrack) {
   switch (kfref->algo()) {
   case TrackBase::undefAlgorithm:
   case TrackBase::ctf:
+  case TrackBase::duplicateMerge:
   case TrackBase::initialStep:
   case TrackBase::lowPtTripletStep:
   case TrackBase::pixelPairStep:

